src/OmniSharp.MSBuild/ProjectFile/PackageReference.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using NuGet.Packaging.Core;
+
+namespace OmniSharp.MSBuild.ProjectFile
+{
+    public class PackageReference : IEquatable<PackageReference>
```

Comment:
nuget has this https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.PackageManagement/Utility/PackageReferenceComparer.cs, maybe there is no need to introduce a class into omnisharp?